src/swipe/swipe.service.ts

```diff
@@ -22,4 +22,22 @@ export const SwipeService = new (class {
 
```

Comment:
In line 20 in swipe-service I think it should be `swipe.submittedBy = submittedByUser` instead of `swipe.submittedBy = targetUser`

Reply:
true, I will change it :)